custom/icds_reports/management/commands/backfill_child_health_monthly.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from datetime import date
+
+from dateutil.rrule import MONTHLY, rrule
+from django.core.management.base import BaseCommand
+from django.db import connections
+
+from corehq.sql_db.connections import get_icds_ucr_db_alias
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        start_date = date(2017, 3, 1)
```

Comment:
this is when teh dashboard starts, but i think there are tables from before this date. on me to figure out the correct date to start with